src/library/dlgtrackinfo.h

```diff
@@ -92,7 +92,7 @@ class DlgTrackInfo : public QDialog, public Ui::DlgTrackInfo {
     TrackPointer m_pLoadedTrack;
     mixxx::BeatsPointer m_pBeatsClone;
     Keys m_keysClone;
-    bool m_trackHasBeatMap;
+    bool m_bpmIsConst;
```

Comment:
I would prefer the term *uniform* instead of *const*. This would also prevent confusion with *const* in programming @Be Any suggestions from a native speaker?

Reply:
I think "constant" is appropriate, but I think it should be fully spelled out as "Constant" instead of abbreviated "Const" to avoid confusion with C++'s `const` keyword.